docs/installation.rst

```diff
@@ -82,55 +73,27 @@ Installing
     python setup.py install
 
 
-Downloading Required Data
-*************************
+Downloading Required Reference Data
+************************************
```

Comment:
```suggestion
.. _downloading-required-ref-data:

Downloading Required Reference Data
************************************
```
This, combined with another suggestion I made earlier in the document, allows you to directly link to this section  from other locations in this page.